src/braket/jobs/hybrid_job.py

```diff
@@ -0,0 +1,333 @@
+import functools
+import importlib.util
+import inspect
+import shutil
+import tempfile
+import warnings
+from logging import Logger, getLogger
+from pathlib import Path
+from types import ModuleType
+from typing import Any, Callable, Dict, Iterable, Optional, Union
```

Comment:
We've now deprecated Python 3.8, so you no longer have to import Callable, Dict and Iterable

Reply:
Exciting, I will update this PR to use 3.9+ syntax